### PR TITLE
LRQA-44335 Update sf failure message generator to avoid empty content

### PR DESCRIPTION
**Pull request by yichenroy**
https://issues.liferay.com/browse/LRQA-44335

### CODE REVIEW

**Comment by yichenroy** [comment]
ci:test:sf

**Comment by yichenroy** [comment]
ci:test:relevant

File: modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/failure/message/generator/SourceFormatFailureMessageGenerator.java [hunk 1]
```diff
@@ -35,7 +35,9 @@ public Element getMessageElement(Build build) {
 
 		int start = consoleText.lastIndexOf(_TOKEN_SOURCE_FORMAT);
 
-		start = consoleText.lastIndexOf(_TOKEN_FORMATTING_ISSUES, start);
+		start = consoleText.lastIndexOf(_TOKEN_FORMAT_SOURCE_ALL, start);
```
Comment (pyoo47) [inline modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/failure/message/generator/SourceFormatFailureMessageGenerator.java]:
```suggestion
		start = consoleText.lastIndexOf(_TOKEN_FORMAT_SOURCE_ALL, start);
		
		if (start == -1) {
		    return null
		}
```

**Comment by yichenroy** [comment]
ci:close
The problem lies in the json body that was printed in stop-current-job, we will make change in jenkins-ee to stop outputting the content instead.

**Pull request closed without merging**